tests/foreman/ui/test_remoteexecution.py

```diff
@@ -639,11 +642,4 @@ def test_positive_configure_cloud_connector(
     logger.debug(f"Repolist>>\n{repolist}\n<<End of repolist")
 
     assert entities.JobInvocation(id=invocation_id).read().status == 0
-    assert 'project-receptor.satellite_receptor_installer' in result
     assert 'Exit status: 0' in result
-    # check that there is one receptor conf file and it's only readable
-    # by the receptor user and root
-    result = target_sat.execute('stat /etc/receptor/*/receptor.conf --format "%a:%U"')
-    assert all(filestats == '400:foreman-proxy' for filestats in result.stdout.strip().split('\n'))
-    result = target_sat.execute('ls -l /etc/receptor/*/receptor.conf | wc -l')
-    assert int(result.stdout.strip()) >= 1
```

Comment:
Same here, we should check that it's really configured rather than just trust the passing result of a job.

Reply:
yes, we should. I will try to find something.